db/migrate/20210609041601_devise_create_users.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
```

Comment:
Based on our convention, email field's type should be `citext` 
![Screen Shot 2021-06-14 at 2 46 32 PM](https://user-images.githubusercontent.com/6076762/121857098-7117d680-cd1f-11eb-8767-aaad76186802.png)